junebug/api.py

```diff
@@ -301,3 +304,33 @@ def get_message_status(self, request, channel_id, message_id):
     @app.route('/health', methods=['GET'])
     def health_status(self, request):
         return response(request, 'health ok', {})
+
+    @app.route('/channels_health', methods=['GET'])
+    @inlineCallbacks
+    def channels_health(self, request):
+        channels_stuck = []
+
+        base_url = "http://%s:%s/api/queues/%s" % (
+            self.amqp_config['hostname'],
+            self.amqp_config.get('api_port', '15672'),
+            self.amqp_config['vhost'].replace('/', '%2F'))
```

Comment:
We should rather urlencode the entire vhost string, rather than just urlencoding the `/` character, otherwise there might be other characters in the vhost name that we might run into that need to be encoded.